### DISCUSSION TITLE
How to connect Hasura GraphQL from Spring Boot App

### DISCUSSION THREAD

**Discussion by bala-ravuthan** [hasura/graphql-engine#8196] [Question] [answered]
How to connect Hasura GraphQL from Spring Boot App?

**Comment by ecthiender** [comment, marked as answer]
You will need a GraphQL client for Java. Using that you would connect to Hasura. This page has various options listed - https://graphql.org/code/#java-kotlin

Check out what is a graphql client here - https://hasura.io/learn/graphql/intro-graphql/graphql-client/
